client/package.json

```diff
@@ -31,6 +31,5 @@
     "build": "react-scripts build",
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
-    "postinstall": "npm run build"
```

Comment:
Correct me if I'm wrong, don't we need this line for Heroku to deploy the client successfully?

Reply:
The `postinstall` command is built into the ROOT directories `package.json` so it should run that which does this. `"postinstall": "npm install --prefix client && npm run build --prefix client"`

Reply:
The `--prefix client` portion is telling Heroku to run these commands inside the `client` folder.

Reply:
Understood, I didn't notice this was the one inside the client directory.